Zilon.Core/Zilon.Core.Tests/MapGenerators/SectorProceduralGeneratorTests.cs

```diff
@@ -33,11 +34,12 @@ public void Create_DifferentMaps_NoExceptions()
             var botPlayer = CreateBotPlayer();
             var generator = CreateGenerator(botPlayer, mapFactory);
             var sectorScheme = CreateSectorScheme();
+            var sectorNode = CreateSectorNode(sectorScheme);
 
             // ACT
             Func<Task> act = async () =>
             {
-                var sector = await generator.GenerateDungeonAsync(sectorScheme).ConfigureAwait(false);
+                var sector = await generator.GenerateAsync(sectorNode).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'sector'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)